safitty/parser.py

```diff
@@ -142,23 +150,29 @@ def save(
     Args:
         storage (Storage): config to save
         path (Union[str, Path]): path to save
+        data_format (str): ``yaml``, ``yml`` or ``json``. If not specified,
+            safitty looks at ``path.suffix``
         encoding (str): Encoding to write file. Default is ``utf-8``
         ensure_ascii (bool): Used for JSON, if True non-ASCII
             characters are escaped in JSON strings.
         indent (int): Used for JSON
     """
     path = Path(path)
 
-    if not is_file_supported(path):
-        raise Exception(f"Unknown file format '{path.suffix}'")
+    if data_format is not None:
+        suffix = data_format
+    else:
+        suffix = path.suffix
+    assert is_file_supported(suffix), \
+        f"Unknown file format '{suffix}'"
```

Comment:
maybe it is better to use:
```suggestion:-1+0
    if not is_file_supported(suffix):
        raise Exception(f"Unknown file format '{suffix}'")
```